modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,205 @@
+The processor passes on the new message as output to the next processor in the flow.
+
+Event processors available to Mule apps include these:
+
+* Operations from modules and connectors. Examples include reading from and
+writing to an external resource and validating data in the Mule app.
++
+Some operations can make client requests to external resources and services
+(including external databases and systems, such as Salesforce, Workday,
+ServiceNow, and many others) and to other Mule apps. Others can perform
+validations or run your custom code, for example. A number of other operations
+are available.
++
+* Core components. Examples include routing data, performing data
+transformations, and handling errors that might occur when processing the event.
++
+Transformers (such as the Transform Message, Set Variable, and others) are
+key to exchanging data between nodes. Transformers allow Mule to convert message
+data in the Mule event to a format that another application or service can read.
++
+Mule also enables content enrichment of messages (through
+xref:target-variables[Target Parameters]) so that you can retrieve
+additional data and attach it to the message.
+
+For more on this topic, see xref:about-components[Core Components] and xref:connectors:index.adoc[Connectors and Modules].
+
+//image::flow1-1.png[Flow1-1]
+
+=== Anypoint Studio and Design Center
+
+You can design Mule apps in Anypoint Studio, in Design Center, or even with
+handwritten XML. Studio and Design Center UIs represent message processors as
+icons within flow components. Studio also represents them as XML elements in a
+Configuration XML.
+
+////
+The example below is a simple flow created in Anypoint Studio's visual editor;
+click the *XML Editor* tab to view the XML of a Mule project that holds the same
+flow.
+
+[.ex]
+=====
+[discrete.view]
+=== Studio Visual Editor
+
+image::simple-flow2.png[simple+flow2]
+
+[discrete.view]
+=== XML Editor
+
+[source,xml,linenums]
+----
+include::{examplesdir}/mule-concepts.xml[]
+----
+=====
+////
+
+For more on this topic, see
+xref:design-center/about-designing-a-mule-application.adoc[Mule Flow Design (Design Center)]
+and xref:7.2@studio:index.adoc[Anypoint Studio].
+
+[[dataweave]]
+== DataWeave
+
+DataWeave is the primary language used for formulating expressions in Mule.
+Within fields of many message processing components, in addition to the
+Transform Message component, you can use DataWeave to access, manipulate, transform, and extract data from the Mule event.
+
+At runtime, Mule evaluates expressions while executing a flow to:
+
+* Extract data needed to process the current message.
+* Set or manipulate a value in the message.
+* Perform an operation on data in the message, application, Mule instance, or server.
+
+For more on this topic, see xref:dataweave.adoc[DataWeave Language].
+
+== Batch Processing
+
+Studio and Design Center support batch jobs, which split messages into
```

Comment:
I think this is confusing, Studio and DC might support batch in their UI but this sounds like they are the ones that feature the processing of batch records, not the runtime.

Reply:
replaced "Studio and Design Center" with "Mule Runtime".

Reply:
replaced Studio and DC with Mule Runtime